microscopium/serve.py

```diff
@@ -267,6 +270,19 @@ def update_table(indices, df, table):
     filtered_df = df.iloc[indices]
     table.source.data = ColumnDataSource(filtered_df).data
 
+def switch_modes_button_group():
```

Comment:
This function should have a docstring. We use numpy style docstrings for this project: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html